main/src/library/File.flix

```diff
@@ -15,6 +15,173 @@
  */
 namespace File {
 
+    ///
+    /// File types.
+    ///
+    pub enum FileType with Eq, Order, ToString {
+        case File,
+        case Directory,
+        case SymLink,
+        case Other
+    }
+
+    ///
+    /// Returns the last access time of the given file `f` in milliseconds.
+    ///
+    pub def accessTime(f: String): Result[Int64, String] & Impure =
+        try {
+            import new java.io.File(String) as newFile;
+            import java.io.File.toPath();
+            import static java.nio.file.Files.readAttributes(##java.nio.file.Path, ##java.lang.Class, Array[##java.nio.file.LinkOption]);
+            import static java.lang.Class.forName(String);
+            import java.nio.file.attribute.BasicFileAttributes.lastAccessTime();
+            import java.nio.file.attribute.FileTime.toMillis();
+
+            let javaFile = newFile(f);
+            let javaPath = toPath(javaFile);
+            let classOfAttributes = forName("java.nio.file.attribute.BasicFileAttributes");
+            let fileAttributes = readAttributes(javaPath, classOfAttributes, []);
```

Comment:
Do you think it would be cleaner to factor out these four lines into a private helper function to reuse among those `*Time` functions?

Reply:
Yes, that would make sense, since they are used multiple times. Avoid code dublication :-)